src/crypto/tls/common.go

```diff
@@ -469,6 +479,20 @@ const (
 	ECDSAWithSHA1 SignatureScheme = 0x0203
 )
 
+// A TLSFlag is the index of the bit in the TLSFlags bit array that should be
```

Comment:
Go comment conventions.
```suggestion
// TLSFlag is the index of a bit in the TLSFlags bit array that should be
```